src/FlightMap/Widgets/CameraPageWidget.qml

```diff
@@ -67,15 +68,21 @@ Column {
         id:             cameraLabel
         text:           _camera ? _camera.modelName : qsTr("Camera")
         visible:        _camera
-        font.pointSize: ScreenTools.smallFontPointSize
+        font.pointSize: ScreenTools.defaultFontPointSize
         anchors.horizontalCenter: parent.horizontalCenter
     }
     QGCLabel {
         text: _camera ? qsTr("Free Space: ") + _camera.storageFreeStr : ""
-        font.pointSize: ScreenTools.smallFontPointSize
+        font.pointSize: ScreenTools.defaultFontPointSize
         anchors.horizontalCenter: parent.horizontalCenter
         visible: _camera && _storageReady
     }
+    QGCLabel {
+        text: _camera ? qsTr("Battery: ") + _camera.batteryRemainingStr : ""
+        font.pointSize: ScreenTools.defaultFontPointSize
+        anchors.horizontalCenter: parent.horizontalCenter
+        visible: _camera && _batteryReady
+    }
```

Comment:
`_batteryReady` already tests for `_camera`
```
_batteryReady:  _camera && _camera.batteryRemaining >= 0
```
Which makes this a bit redundant:
```
visible: _camera && _batteryReady
```
I would normally merge and deal with it later but I'm not planning on working on this for a while.

Reply:
nice 
the same holds for _storageReady, used just few lines up the same way. I grabbed the occasion to also correct this